flow/raster_cache.cc

```diff
@@ -70,23 +79,16 @@ static bool IsPictureWorthRasterizing(SkPicture* picture,
 
 RasterCacheResult RasterizePicture(SkPicture* picture,
                                    GrContext* context,
-                                   const MatrixDecomposition& matrix,
+                                   const SkMatrix& ctm,
```

Comment:
Do we still need `MatrixDecomposition`?

Reply:
I mean: can we remove the class now?

Reply:
Scenic node hierarchies are updated based on the results of the matrix decomposition. That part is still in flow currently though I am planning to move it out.

Reply:
I think that we no longer needs MatrixDecomposition. My original plan was to remove that class later so we can keep this PR small. Should I remove it in this PR?

Reply:
Ah, just saw Chinmay's comment. Then we can only remove it after Scenic is no longer using it?

Reply:
You could move it into an OS_FUCHSIA block but the unit tests for that class are only run by the engine bots. So I would keep it as is for now.

Reply:
I see. I'll then also merge this PR as is.